src/GridList/GridTile.js

```diff
@@ -111,6 +112,10 @@ class GridTile extends Component {
      */
     titleBackground: PropTypes.string,
     /**
+     * Style used for title color.
+     */
+    titleColor: PropTypes.string,
```

Comment:
I think that it would be better to add a `titleStyle` property instead of this very specific one.